armsrc/hitagS.c

```diff
@@ -367,19 +377,23 @@ static void hts_init_clock(void) {
         AT91C_TC_ABETRG |                   // TIOA is used as an external trigger
         AT91C_TC_LDRA_FALLING;              // load RA on on falling edge
 
+    // TC2: Capture mode, clock source = MCK/32 (TIMER_CLOCK3), no triggers
+    AT91C_BASE_TC2->TC_CMR = AT91C_TC_CLKS_TIMER_DIV3_CLOCK;
+
     // Enable and reset counters
-    AT91C_BASE_TC0->TC_CCR = AT91C_TC_CLKEN | AT91C_TC_SWTRG;
-    AT91C_BASE_TC1->TC_CCR = AT91C_TC_CLKEN | AT91C_TC_SWTRG;
+    AT91C_BASE_TC0->TC_CCR = AT91C_TC_CLKEN;
+    AT91C_BASE_TC1->TC_CCR = AT91C_TC_CLKEN;
```

Comment:
there is a bug mentioned in the errata of the arm chip,   use previous declaration.

Reply:
I didn't find any Timer Counter info in the errata section. If you are referring to the reset, the three timer counters are reset together using AT91C_BASE_TCB->TCB_BCR = 1

Reply:
No.

And then I will close this PR.

Reply:
Yes,  that would be the ones.

Reply:
Great! Then the reset is already handled together using the SYNC signal (AT91C_BASE_TCB->TCB_BCR = 1) which generates a software trigger for each channel simultaneously. I think it's ready to be merged.

Reply:
No,  
Revert back the  `= AT91C_TC_CLKEN | AT91C_TC_SWTRG`  parts.

Reply:
But why? The reset is properly handled by the SYNC signal. Is the redundant code for bug-proofing in case someone copy/paste it elsewhere, or just to maintain consistent code style?
Anyway, I will revert it back.